### PR TITLE
[frontend/backend] Filters bug fixes 2 (#2686)

### PR DESCRIPTION
**Pull request by Archidoit**

### CODE REVIEW

File: opencti-platform/opencti-front/src/components/TasksFilterValueContainer.tsx [hunk 1]
```diff
@@ -21,6 +48,26 @@ const TasksFilterValueContainer = ({ filters }: { filters: FilterGroup }) => {
...
+                <InformationOutline
+                  fontSize="small"
+                  color="secondary"
+                  style={{ cursor: 'default' }}
+                />
+              </Tooltip>
+            </>
+          }
+        />
+      }
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/components/TasksFilterValueContainer.tsx]: On a second thought, we might as well remove this as we already convert it on the fly to new format. This information has no effect on the user, he does not care and won't do anything about it.

what do you think @Archidoit ?

Comment (Archidoit) [reply opencti-platform/opencti-front/src/components/TasksFilterValueContainer.tsx]: It was to warn the user this filters won't be accessible anymore in 1 or 2 versions

Comment (Archidoit) [reply opencti-platform/opencti-front/src/components/TasksFilterValueContainer.tsx]: But indeed we can remove this chip, the 2 solutions are ok

Comment (labo-flg) [reply opencti-platform/opencti-front/src/components/TasksFilterValueContainer.tsx]: ok, I do it

**Review by labo-flg — APPROVED** [review]

**Pull request merged**